docs/sources/whatsnew/whats-new-in-v10-0.md

```diff
@@ -20,6 +20,35 @@ Welcome to Grafana 10.0! Read on to learn about changes to search and navigation
 Description. Include an overview of the feature and problem it solves, and where to learn more (like a link to the docs).
 > **Note:** You must use relative references when linking to docs within the Grafana repo. Please do not use absolute URLs. For more information about relrefs, refer to [Links and references](/docs/writers-toolkit/writing-guide/references/).-->
 
+## Authentication and authorization
+
+### Role-based access control is always enabled
+
+[Role-based access control (RBAC)](({{< relref "../administration/roles-and-permissions/access-control/" >}})) is now always enabled and you can't disable it anymore using configuration option in Grafana.
+
+We understand that this may affect some users who have relied on the ability to disable role-based access control (RBAC) in the past. However, we believe that this change is necessary to ensure the best possible security and user experience for our community and customers.
```

Comment:
nit: you already define RBAC in the previous paragraph.

```suggestion
We understand that this may affect some users who have relied on the ability to disable RBAC in the past. However, we believe that this change is necessary to ensure the best possible security and user experience for our community and customers.
```